### PR TITLE
[Showcase] How to build up your own Seldon Server Environment by PrimeHub Images.

### PR DESCRIPTION
**Pull request by LiuYuWei**
Why:
    1. We can build Seldon server docker images in PrimeHub images.
How:
    1. Add the base version Dockerfile.
    2. Modify readme file content.

### CODE REVIEW

File: custom-seldon-server/README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@
 2. pytorch
 3. sklearn
 4. huggingface
+5. base
```
Comment (jimmyliao) [inline custom-seldon-server/README.md:17]: Curious about base is used to?

File: custom-seldon-server/docker/base/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# From python docker image
+FROM python:3.7-slim
+WORKDIR /app
+
+# Install seldon-core
+RUN pip install seldon-core==1.13.1 werkzeug==2.0.3
```
Comment (jimmyliao) [inline custom-seldon-server/docker/base/Dockerfile:6]: As far as I know werkzeug is WSGI lib, how we use with this showcase?

**Review by jimmyliao — COMMENTED** [review]
Leave some comments, please check.

File: custom-seldon-server/README.md (hunk 1 above)
Comment (LiuYuWei) [reply custom-seldon-server/README.md:17]: The base custom Seldon server uses to build up their custom Seldon server environment by themselves in PrimeHub Images.

File: custom-seldon-server/docker/base/Dockerfile (hunk 2 above)
Comment (LiuYuWei) [reply custom-seldon-server/docker/base/Dockerfile:6]: When I use the `seldon-core` python package, I get the error that needs to install werkzeug. I think seldon-core needs to use the WSGI lib to do something.

**Comment by LiuYuWei** [comment]
Note: Finish solving the conflicts

File: custom-seldon-server/docker/base/Dockerfile (hunk 2 above)
Comment (jimmyliao) [reply custom-seldon-server/docker/base/Dockerfile:6]: got it

**Pull request merged**